metric/prometheus.go

```diff
@@ -179,3 +194,10 @@ func RemoveInvalidChars(name string) string {
 	}
 	return string(result)
 }
+
+func AddConstLabels(labels prometheus.Labels, constLabels prometheus.Labels) prometheus.Labels {
+	for k, v := range constLabels {
+		labels[k] = v
+	}
```

Comment:
should check if a label exists both in `labels` and `constLabels` and report the error.

Reply:
updated